app/code/Magento/AdvancedSearch/Model/ResourceModel/Index.php

```diff
@@ -150,6 +152,63 @@ public function getPriceIndexData($productIds, $storeId)
         return $priceProductsIndexData[$websiteId];
     }
 
+    /**
+     * Return array of inventory data products
+     *
+     * @param null|array $productIds
+     * @return array
+     * @since 100.1.0
+     */
+    protected function _getCatalogProductInventoryData($productIds = null)
```

Comment:
Should be private function 
 `_getCatalogProductInventoryData` change to  `getCatalogProductInventoryData` as Magento standard code.